src/command_modules/azure-cli-network/azure/cli/command_modules/network/_help.py

```diff
@@ -667,11 +672,26 @@
 helps['network asg'] = """
     type: group
     short-summary: Manage application security groups.
+    long-summary: You can configure network security as a natural extension of an application’s structure, ASG allows
```

Comment:
Use the YAML text block `>` for lengthy descriptions so they can be broken into multiple lines.

Lengthy, suggested rewrite:

```
ASG allows grouping of virtual machines underneath a single network security policy.
An ASG can be either the source or destination in a network security group rule. 
For more information...
```